experimental/README.md

```diff
@@ -0,0 +1,39 @@
+<!--
+  SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Experimental Examples
+
+## What is an Experimental Example?
+
+Experimental examples are sample code and deployments for RAG pipelines that are not tested by NVIDIA personnel.
+
+## Inventory
```

Comment:
We have removed it. Can we remove this file in this release?

Reply:
Done. Added reference for streaming-ingest-RAG example.